src/freenet/pluginmanager/OfficialPlugins.java

```diff
@@ -110,7 +110,7 @@ public OfficialPlugins() {
 					.inGroup("communication")
 					.minimumVersion(18)
```

Comment:
Why not increment this?
Disclaimer: i haven't looked up the purpose of this value!
So check how previous releases by e.g. @Thynix  / toad_ did it.

Reply:
Also while you're at it you can increase the value of the minimumVersion() and of the USK at the below testing version of WoT.

Reply:
If I increment the minimumVersion, it will block startup if the key isn’t accessible (i.e. because the node does not have connections yet).

Reply:
Will it only block startup of WoT? That would be OK I suppose if the WoT build fixes significant issues - and in fact build0019 does fix a bug which can block startup of WoT for like half an hour (if not hours, the half an hour is only what I have observed), so nothing would be lost anyway.

Reply:
IIRC it would block startup of fproxy — but recommendedVersion(19) won’t.

Reply:
> IIRC it would block startup of fproxy — but recommendedVersion(19) won’t.

Oh ok nice, then if you or @Thynix validate this to be correct then I would say we go for recommendedVersion(). But please do validate it somehow or otherwise do as previous RMs did as this stuff could have dangerous results like the one you just mentioned.

Reply:
the first step is a testing release :)

Reply:
(re version: I actually have notes on my freenet-release-management website which say that)

Reply:
Thanks. The WoT build has been released to the testing channel for a whole year, does it really need more testing?

Reply:
Oh I see you already marked it as recommended. Thanks.
Please do the same for the *Testing channel.